### PR TITLE
sites list: stripped it down

### PR DESCRIPTION
**Pull request by rjbs**

### CODE REVIEW

File: src/SITES.html [hunk 1]
```diff
@@ -15,50 +15,31 @@ <h1>
...
 <p>
-    [% cpan_stats.mirrors.count | comma %] registered sites around the world
-    make up the <em>N</em> part of CPAN (the Network), find the full list
-    below.
-</p>
-<p>
-    The master CPAN site is hosted by <a href=
-    "http://www.netactuate.com/">NetActuate</a> in Los Angeles, California.
+    For many years, CPAN worked because the central respository was mirrored to
+    many sites around the world.  In the times of a slower global Internet,
```
Comment (rspier) [inline src/SITES.html:19]: times -> time

File: src/SITES.html [hunk 2]
```diff
@@ -15,50 +15,31 @@ <h1>
...
 </p>
-
-<h2>Mirror status</h2>
-
 <p>
-<a href="http://mirrors.cpan.org/">http://mirrors.cpan.org/</a> keeps
-track of the status of the mirrors below.
+    The master CPAN site is hosted by <a href=
+    "http://www.netactuate.com/">NetActuate</a> in Los Angeles, California.
 </p>
```
Comment (rspier) [inline src/SITES.html:38]: Should we note that the CDN is backed by Fastly?

File: src/SITES.html [hunk 3]
```diff
@@ -15,50 +15,31 @@ <h1>
...
-    [% cpan_stats.mirrors.count | comma %] registered sites around the world
-    make up the <em>N</em> part of CPAN (the Network), find the full list
-    below.
-</p>
-<p>
-    The master CPAN site is hosted by <a href=
-    "http://www.netactuate.com/">NetActuate</a> in Los Angeles, California.
+    For many years, CPAN worked because the central respository was mirrored to
+    many sites around the world.  In the times of a slower global Internet,
+    these sites provided most Perl users with fast access to thousands of
```
Comment (rspier) [inline src/SITES.html:20]: sites -> mirrors

File: src/SITES.html [hunk 4]
```diff
@@ -15,50 +15,31 @@ <h1>
...
 </h1>
 <p>
-    [% cpan_stats.mirrors.count | comma %] registered sites around the world
-    make up the <em>N</em> part of CPAN (the Network), find the full list
-    below.
-</p>
-<p>
-    The master CPAN site is hosted by <a href=
-    "http://www.netactuate.com/">NetActuate</a> in Los Angeles, California.
+    For many years, CPAN worked because the central respository was mirrored to
```
Comment (rspier) [inline src/SITES.html:18]: "worked" is weird here.  "was made more successful" ?

**Review by rspier — COMMENTED** [review]
A few minor things, but I'm basically ok with this as is.  I'd merge it today.  We can always refine later.

@abh What do you think?

**Comment by rjbs** [comment]
I added a commit to address some of this.  Any further edits, I leave to you.

**Comment by rspier** [comment]
I'm good with shipping this as is.  Now or later?

**Comment by abh** [comment]
> I'm good with shipping this as is. Now or later?

Did the other announcements go out?

add something to the Makefile to remove the SITES-list.html file? (And/or a change in the other repository to not make it be produced if that’s where that happens).

**Comment by rjbs** [comment]
I have not sent emails.  I'm giving Andreas a few days to weigh in, although I won't wait indefinitely.

**Pull request merged**